app/models/application_qualification.rb

```diff
@@ -59,6 +59,7 @@ class ApplicationQualification < ApplicationRecord
 
   validates :qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
   validates :non_uk_qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
+  validates :enic_reason, presence: true, on: :create
```

Comment:
Should we also add a spec for this validation on create only? 

Something like `it { should validate_presence_of(:enic_reason).on(:create) }`
https://github.com/thoughtbot/shoulda-matchers/blob/main/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb#L115